app/src/main/res/values-cs/strings.xml

```diff
@@ -41,6 +41,7 @@
     <!-- Settings -->
     <string name="group_subsequent_calls">Seskupte další hovory se stejným číslem v protokolu hovorů</string>
     <string name="open_dialpad_by_default">Open the dialpad by default when the app opens</string>
+    <string name="use_proximity_sensor">Turn off screen during calls with proximity sensor</string>
```

Comment:
change it to "Disable proximity sensor during calls" please

Reply:
Changed the string and renamed all the variables to reflect the toggled semantics.